bioio_base/image_container.py

```diff
@@ -129,3 +129,8 @@ def scale(self) -> Scale:
     @abstractmethod
     def time_interval(self) -> TimeInterval:
         pass
+
+    @property
+    @abstractmethod
+    def manifest(self) -> Dict[str, Any]:
```

Comment:
perhaps `parsed_metadata`? especially for .zarr files `.manifest` feels weird

Reply:
> embedded_metadata

I'm fine with that too - it is all embedded and parsed metadata so 🤷

Reply:
Can we have a better type than Dict[str, Any]?  isn't this supposed to be a known collection of things?  like a DataClass or named tuple or something?